src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -10,11 +10,30 @@
 
 namespace Microsoft.ML.Transforms.Text
 {
-
     /// <summary>
-    /// Produces a bag of counts of ngrams (sequences of consecutive words) in a given text.
-    /// It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="ITransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Text](xref:Microsoft.ML.Data.TextDataViewType) |
+    /// | Output column data type | Vector of known-size of <xref:System.Single> |
+    ///
+    /// The resulting <xref:Microsoft.ML.ITransformer> creates a new column, named as specified in the output column name parameters, and
+    /// produces a vector of counts of n-grams (sequences of n consecutive words) from a given data.
+    /// It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag.
+    ///
+    /// <xref:Microsoft.ML.Transforms.Text.WordBagEstimator> is different from <xref:Microsoft.ML.Transforms.Text.NgramExtractingEstimator>
+    /// in a way that the former takes tokenizes text internally while the latter takes tokenized text as input.
```

Comment:
>a way that [](start = 11, length = 10)

same comment as in the extension: "in that the former ... while the latter ...."